src/transformers/models/beit/modeling_beit.py

```diff
@@ -702,7 +702,8 @@ def forward(
         pooled_output = self.pooler(sequence_output) if self.pooler is not None else None
 
         if not return_dict:
-            return (sequence_output, pooled_output) + encoder_outputs[1:]
+            head_outputs = (sequence_output, pooled_output) if pooled_output is not None else (sequence_output,)
```

Comment:
@NielsRogge we should not pass `None` values here as this would not pass an equivalence check in case the pooling layer is not added

Reply:
Thanks for fixing!